scripts/ironic/playbooks/ironic-pre-config.yml

```diff
@@ -0,0 +1,118 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+- name: Pre install config for ironic
+  hosts: localhost
+  tasks:
+    - name: Create ironic network configurations
+      blockinfile:
+        dest: /etc/network/interfaces.d/osa_ironic_interfaces.cfg
```

Comment:
This file could just be in the code tree and copied over, using an ansible task with this content is overcomplicating things.

Reply:
We'll address this in a future version.